sentences/es/media_player_HassMediaNext.yaml

```diff
@@ -3,6 +3,7 @@ intents:
   HassMediaNext:
     data:
       - sentences:
-          - "[<reproduce>] ([<pista> ]siguiente|siguiente <pista>) [en|para] <name>"
+          - "[<reproduce>] [el|la] ([<pista> ]siguiente|siguiente <pista>) [en|para] <name>"
```

Comment:
`para`? That seems weird, I can't think on a grammatically correct way of using `para` in that position.

Reply:
nor there are any examples of that in the tests